### ISSUE TITLE
War Lion stats 

### ISSUE THREAD

**Issue by Exsurge1993** [WarEmu/WarBugs#19513]
Hello

I did some testing on my White Lion with the Guardian Path to determine the stat increases you get by putting mastery points in Path of the Guardian accoring to the wiki:

Pet wounds, toughness, weapon skill, strength, and initiative are reduced, however:
Points in Hunter tree grant strength and weapon skill to your pet
Points in the Axeman tree grant toughness and initiative to your pet
Points in the Guardian tree grant wounds, toughness, weapon skill, strength, and initiative to your pet

Turns out there isnt ANY stat increase. The warlion does and takes the exact same damage with the Path of the Guardian maxed out and with no points at all invested. That combined with the fact War Lions are not affected by 99.9% of group buffs (i dont know if this is intended or not) makes them a terribly weak ball of fur. 

Please consider fixing the stat increase with mastery points on warlions as you would be re enabling builds that are not the tactic Loner. As I personaly would love to play the pet specc.

**Comment by Exsurge1993** [comment]
Hey,

Do I now have to get other White Lions to support this topic or is this already on a list  somewhere somehow ? Realy dont want to be demanding but I dont know how this works.

**Comment by Exsurge1993** [comment]
Hello 
Could someone please advice me on how to progress this topic, should I be rallying support ?
Something like: we have seen this problem and will eventually get to it would sufice for me to happily continue playing my WL.

**Comment by CrimsonFenris** [comment]
Afaik, your pet stats are not affected by mastery tree points, but by you own stats ; depending on used training (stance), your pet willl get a bonus to its stats, depending on your own gear bonuses.

So if you use Trained to kill, the pet should be granted your gear bonus into Strength and Weapon Skill.

You can check that by selecting your pet and typing ".getstat" ; used stance will modify its stats.

I dont think that mastery points really increase pet skills **base damage**, indeed, but the stat contribution from your character should work as intended.

**Comment by Exsurge1993** [comment]
> Afaik, your pet stats are not affected by mastery tree points, but by you own stats ; depending on used training (stance), your pet willl get a bonus to its stats, depending on your own gear bonuses.
> 
> So if you use Trained to kill, the pet should be granted your gear bonus into Strength and Weapon Skill.
> 
> You can check that by selecting your pet and typing ".getstat" ; used stance will modify its stats.
> 
> I dont think that mastery points really increase pet skills **base damage**, indeed, but the stat contribution from your character should work as intended.

That is correct, thank you for your observatin. Trained to kill and the other "aspects" work as Intended. This is not about that aspect of the lion stats tho as my lion just dies with his 700 added strength and some weaponskill true trained to kill. And the damage is still low aswell. There is no way to make the pet specc viable again without fixing the stat increase from Mastery Points on the War Lion as there is no way to increase there defense without swaping to Trained to Threaten which I do to recover the lion but this doesnt realy work due to only max hp chaning on swaping so the lion is % on even lower HP then befor. I tried everything I run bgs with Furious Mending and a healing power + willpower Liniment and he just dies next to 1 chopper. There is actually nothing we can do currently to make the pet do damage and not just flop.

**Comment by Exsurge1993** [comment]
Afaik, your pet stats are not affected by mastery tree points, but by you own stats ; depending on used training (stance), your pet willl get a bonus to its stats, depending on your own gear bonuses.

This was changed a while ago this is the change :
Pet wounds, toughness, weapon skill, strength, and initiative are reduced, however:
Points in Hunter tree grant strength and weapon skill to your pet
Points in the Axeman tree grant toughness and initiative to your pet
Points in the Guardian tree grant wounds, toughness, weapon skill, strength, and initiative to your pet

The reduce in base stat was suppoused to be counteracted by the increase of pet stats we should get by putting points in the diffrent trees, as seen above.
Right now we dont have anything no base stats and no stat increases true mastery points. What you are saying with the aspects is correct but never the problem  you can clearly read in the patch notes that these are suppoused to be diffrent stat increases. Please just test 1 Scenario with a Copper on the enemy side and it becomes so clearly apparant what the problem is. Meanwhile the pet gcd got fixed ... cant help but to be frustrated.
Maybe I need to reiterate, I am not looking for more damage on the lion even tho he should have more beceaus of the base stat change I am mainly looking for it not to die in 2 seconds while scirmishing or getting targeted. Also they can not be target by party buff like you where able to on Retail. They are so worthless compaired to what they used to be : the best designed pet class in an mmo yet
